include/mxnet/runtime/ffi_helper.h

```diff
@@ -99,6 +99,24 @@ class Integer: public ObjectRef {
   MXNET_DEFINE_OBJECT_REF_METHODS(Integer, ObjectRef, IntegerObj)
 };
 
+class FloatObj: public Object {
+ public:
+  double value;
+  static constexpr const uint32_t _type_index = TypeIndex::kFloat;
+  static constexpr const char* _type_key = "MXNet.Float";
+  MXNET_DECLARE_FINAL_OBJECT_INFO(FloatObj, Object)
+};
+
+class Float: public ObjectRef {
+ public:
+  explicit Float(double value,
+                 ObjectPtr<IntegerObj>&& data = make_object<IntegerObj>()) {
```

Comment:
`ObjectPtr<FloatObj>` and `make_object<FloatObj>`

Reply:
Done